sql-insert/insert-languages.sql

```diff
@@ -0,0 +1,7 @@
+insert into "languages" ("name")
+values ('HTML'), ('CSS'), ('JavaScript');
+
+select *
+  from "languages";
```

Comment:
Same here, should be using a `returning`

Reply:
is `returning` being used because we want to only return the rows and values of what we added?

Reply:
@sharon-tieu Exactly.